rise/static/main.js

```diff
@@ -483,13 +531,17 @@ function revealMode() {
     buttonHelp();
     $('#maintoolbar').addClass('reveal_tagging');
   } else {
+    var current_cell_index = reveal_cell_index(IPython.notebook);
     Remover(config);
     setupKeys("notebook_mode");
     $('#exit_b').remove();
     $('#help_b').remove();
     $('#maintoolbar').removeClass('reveal_tagging');
     // Workaround... should be a better solution. Need to investigate codemirror
     fixCellHeight();
+    // select and focus on current cell
+    IPython.notebook.select(current_cell_index);
+    IPython.notebook.get_selected_cell().ensure_focused();
```

Comment:
I don't thing this line is necessary... selecting the cell by index in the previous should be enough, I think.